Source/Fuse.Common/Internal/iOSSystemFont.uno

```diff
@@ -14,16 +14,34 @@ namespace Fuse.Internal
 		static HashSet<string> _families;
 		static Dictionary<string, string> _fontPaths;
 		static readonly string[] _weightNames = new string[]
-			{ "UltraLight", "Thin", "Light", "Regular", "Medium", "SemiBold", "Bold", "Heavy", "Black" };
+			{ "UltraLight", "Thin", "Light", "Regular", "Medium", "Semibold", "Bold", "Heavy", "Black" };
 
 		public static List<FontFaceDescriptor> Default
 		{
 			get
 			{
-				var descriptor = GetDefaultUIFontDescriptor();
-				var descriptors = GetFallbackUIFontDescriptors(descriptor);
-				var fontNames = GetFontNamesFromUIFontDescriptors(descriptors);
-				return GetDescriptorsFromFontNames(fontNames);
+				#if defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0
```

Comment:
These are C preprocessor macros and won't work in Uno code. Since we run-time check for iOS 13 on the line below, I think we can just remove the `#if..#endif` stuff here.